modules/upgrade/pages/db-migration-13.adoc

```diff
@@ -0,0 +1,124 @@
+[[db-migration-13]]
+= Database Migration from Version 12 to 13
+
+This section covers upgrading the PostgreSQL database from version{nbsp}12 to version{nbsp}13.
+If you are already using PostgreSQL 13, you do not need to perform this migration.
+If you are using version 10, see xref:upgrade:db-migration-12.adoc[].
+If you are using an older version, such as version 9.6, see xref:upgrade:db-migration-10.adoc[].
+
+If you want to upgrade to the latest {productname} version, you must be using PostgreSQL version 12 or 13, depending on the underlying operating system. When running on SLES 15 SP3, version 13 should be used. When running on Leap 15.2, version 12 should be used.
+
+
+
+////
+== Background Information
+
+
+* {productname} 3.1.4, SLES 12 SP3, Postgres 9.6
+* {productname} 3.1.11, SLES 12 SP3, Postgres 9.6
+* {productname} 3.2.10, SLES 12 SP3, Postgres 9.6
+* {productname} 3.2.10, SLES 12 SP3, Postgres 10
+* {productname} 3.2.10, SLES 12 SP4, Postgres 10
+* {productname} 4.0.0, SLES 15 SP1, Postgres 10
+* {productname} 4.1.0, SLES 15 SP2, Postgres 12
+* {productname} 4.2.0, SLES 15 SP3, Postgres 13
+
```

Comment:
I feel like this is actually useful information for the reader. Can we turn it into a table perhaps?

Reply:
I am not that familiar with asciidoc and actually I don't know how to render it to test it.

How would a table look like in asciidoc?

Reply:
Are we talking about this?

```
[options="header"]
|=================================================
| SUSE Manager version | SLES version | Postgresql version
| {productname} 3.1.4 | SLES 12 SP3 | Postgres 9.6                                
| {productname} 3.1.11 | SLES 12 SP3 | Postgres 9.6                               
| {productname} 3.2.10 | SLES 12 SP3 | Postgres 9.6                               
| {productname} 3.2.10 | SLES 12 SP3 | Postgres 10                                
| {productname} 3.2.10 | SLES 12 SP4 | Postgres 10                                
| {productname} 4.0.0 | SLES 15 SP1 | Postgres 10                                 
| {productname} 4.1.0 | SLES 15 SP2 | Postgres 12                                 
| {productname} 4.2.0 | SLES 15 SP3 | Postgres 13
|=================================================
```

Reply:
You can look at the install guide requirements chapter, the client cfg guide, or the reference guide to see examples, but this is the basic syntax:

```
[[ref-id]]
.Title
[cols="1,1,1", options="header"]
|===
| Column 1                  | Column 2                 | Column 3
| Content in column 1 | Content in column 2 | Content in column 3
|=== 
```

All good if you don't want to do this, though, just raise a new issue and I'll sort it out :)

Reply:
Should the ref-id be "ref-id" or be some kind of id? If so, which?

I have this now

`
////                                                                            
== Background Information                                                       
[[ref-id]]                                                                      
[cols="1,1,1", options="header"]                                                
|=================================================                              
| {productname} version | SLES version | Postgresql version                     
| {productname} 3.1.4 | SLES 12 SP3 | Postgres 9.6                                
| {productname} 3.1.11 | SLES 12 SP3 | Postgres 9.6                               
| {productname} 3.2.10 | SLES 12 SP3 | Postgres 9.6                               
| {productname} 3.2.10 | SLES 12 SP3 | Postgres 10                                
| {productname} 3.2.10 | SLES 12 SP4 | Postgres 10                                
| {productname} 4.0.0 | SLES 15 SP1 | Postgres 10                                 
| {productname} 4.1.0 | SLES 15 SP2 | Postgres 12                                 
| {productname} 4.2.0 | SLES 15 SP3 | Postgres 13                               
|=================================================                              
                                                                                
                                                                                
////
`